docs/supported_ops.md

```diff
@@ -9,7 +9,7 @@ support all data types. The RAPIDS Accelerator for Apache Spark has further
 restrictions on what types are supported for processing. This tries
 to document what operations are supported and what data types each operation supports.
 Because Apache Spark is under active development too and this document was generated
-against version 3.0.0 of Spark. Most of this should still
+against version 3.0.1 of Spark. Most of this should still
```

Comment:
The default is still to use 3.0.0 for building the docs.  What happened to change it to 3.0.1 when the build itself did not change?

Reply:
My bad. I ran `mvn verify` with `Dspark.version=3.0.1` to update the doc.